pom.xml

```diff
@@ -51,6 +51,16 @@
       <version>1.2.16</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>2.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <version>2.4</version>
+    </dependency>
```

Comment:
I don't see log4j being used anywhere in this change.  Can these be not added?

Reply:
I added them when I was logging stuff, will remove.